VERSION ""


NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    BA_DEF_DEF_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_SGTYPE_
    BA_SGTYPE_
    SIG_TYPE_REF_
    VAL_TABLE_
    SIG_GROUP_
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    BO_TX_BU_
    BA_DEF_REL_
    BA_REL_
    BA_DEF_DEF_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_:

BU_: XXX

BO_ 566 STEER_ANGLE_SENSOR: 8 XXX
 SG_ STEER_ANGLE : 0|16@1- (-0.5,0) [0|1800] "degree" XXX

BO_ 753 STEER_MOMENT_SENSOR: 8 XXX
 SG_ STEER_MOMENT : 16|32@1- (1,0) [0|65535] "" XXX

BO_ 528 GAS_PEDAL: 8 XXX
 SG_ GAS_PEDAL : 24|8@0+ (1,0) [0|255] "" XXX

BO_ 536 GEARBOX: 8 XXX
 SG_ GEAR_SHIFTER : 16|3@0+ (1,0) [0|255] "" XXX


